src/shaders/gfx/Shaders.conf

```diff
@@ -1,13 +1,13 @@
-group = default-shaders
+group = gfx-shaders
 
 # To avoid a needless copy when the sources get added to GL::Shader
 nullTerminated=true
 
 [file]
-filename = depth.vert
+filename = ../gfx_batch/depth.vert
 
 [file]
-filename = depth.frag
+filename = ../gfx_batch/depth.frag
```

Comment:
I don't think the shaders are used for anything here anymore. No need to list them.